app/src/main/java/com/arygm/quickfix/model/messaging/Message.kt

```diff
@@ -2,9 +2,9 @@ package com.arygm.quickfix.model.messaging
 
 import com.google.firebase.Timestamp
 
-class Message(
-    val messageId: String,
-    val senderId: String,
-    val content: String,
-    val timestamp: Timestamp
+data class Message(
+    val messageId: String = "", // Default value
+    val senderId: String = "", // Default value
+    val content: String = "", // Default value
+    val timestamp: Timestamp = Timestamp.now() // Default value
```

Comment:
Would be good here to have a additional field to see if the message is read or not (so that when for example 2 messages are not read on the chat field, it shows a small notification on the dashboard screen, something like this:
![image](https://github.com/user-attachments/assets/b2e9f1c9-8603-4c5d-9ff6-4bcfb8d74a6c)


I'll add it on a future PR it's not something required asap